Amplitude-Swift.xcodeproj/project.xcworkspace/contents.xcworkspacedata

```diff
@@ -4,4 +4,4 @@
    <FileRef
       location = "self:">
    </FileRef>
-</Workspace>
\ No newline at end of file
+</Workspace>
```

Comment:
I have this newline and "remoteGlobalIDString" in `project.pbxproj` with quotation marks added by Xcode automatically every time. It's nice to see them committed.